ext/pdo/pdo_dbh.c

```diff
@@ -38,6 +38,51 @@
 
 static int pdo_dbh_attribute_set(pdo_dbh_t *dbh, long attr, zval *value TSRMLS_DC);
 
+PDO_API zval *pdo_throw_exception_ex(zend_class_entry *exception_ce, pdo_error_type *pdo_err, long code TSRMLS_DC, const char *format, ...) /* {{{ */
```

Comment:
Well, zend_throw_exception_ex was extendable so I wanted to keep that option, but your right this is possibly not required or wanted in a library where the purpose is to unify driver interaction.